pkg/interface/src/views/apps/chat/components/lib/chat-window.tsx

```diff
@@ -1,25 +1,64 @@
 import React, { Component, Fragment } from "react";
+import { Virtuoso as VirtualList } from 'react-virtuoso';
 
 import { ChatMessage } from './chat-message';
-import { ChatScrollContainer } from "./chat-scroll-container";
 import { UnreadNotice } from "./unread-notice";
 import { ResubscribeElement } from "./resubscribe-element";
 import { BacklogElement } from "./backlog-element";
+import { Envelope, IMessage } from "~/types/chat-update";
+import { RouteComponentProps } from "react-router-dom";
+import { Patp, Path } from "~/types/noun";
+import { Contacts } from "~/types/contact-update";
+import { Association } from "~/types/metadata-update";
+import { Group } from "~/types/group-update";
+import GlobalApi from "~/logic/api/global";
+
+// This component gets passed an envelopes prop with 20 envelopes
+// Why? No one knows.
```

Comment:
this is because the initial chat-view subscription returns 20 envelopes for all chats. More are requested when the chat is selected

Reply:
Oh ok, so this is an arvo feature not a landscape one